modules/plugins/randompictures.go

```diff
@@ -85,11 +85,11 @@ func (rp *RandomPictures) Init(session *discordgo.Session) {
 			cursor, err := rethink.Table("randompictures_sources").Run(helpers.GetDB())
 			helpers.Relax(err)
 			err = cursor.All(&rpSources)
-			helpers.Relax(err)
-			if err != nil && err != rethink.ErrEmptyResult {
+			if err == rethink.ErrEmptyResult {
 				raven.CaptureError(fmt.Errorf("%#v", err), map[string]string{})
 				continue
 			}
+			helpers.Relax(err)
```

Comment:
ah actually, we don't want to send EmptyResult errors to sentry.
if ErrEmptyResult: continue (plus time.Sleep for about 30 seconds to prevent DB requests like every second)
helpers.Relax to send other errors to sentry and stop the whole goroutine